ecs-agent/daemonimages/csidriver/Makefile

```diff
@@ -1,27 +1,33 @@
-CSI_DRIVER_VERSION="v$(shell cat ./container/VERSION)"
-GO111MODULE=on
-GOPATH=$(shell go env GOPATH)
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+.PHONY: all tar image mockgen clean
+
+CSI_DRIVER_VERSION="v$(shell cat CSI_DRIVER_VERSION)"
 BUILD_DATE=$(shell date -u -Iseconds)
+GO_VERSION=$(shell cat ../../../GO_VERSION)
 
-OS?=linux
-ARCH?=amd64
+all: tar
 
-.PHONY: bin/csi-driver
-bin/csi-driver:
```

Comment:
Should we leave this make target here in case developers want to build a local binary?

Reply:
personally I'd prefer developers just default to the dockerfile to avoid complicated logic around handling different local developer OS and cpu architectures

Reply:
done!